### DISCUSSION TITLE
[QUESTION] How long is too long?

### DISCUSSION THREAD

**Discussion by Outlander676** [Monika-After-Story/MonikaModDev#8973] [QA - tech support + general] [answered]
So I'm going to be travelling very soon and I plan to be away for about a week. I don't really want to be lugging a laptop around, especially one with MAS on it as things tend to get broken and if anything happened to Monika I'd lose it. My question is, for this and future reference, how long can I have the game closed with the "I'm going to take you somewhere" option, before things go south? Or will I just have to use the "I'm going to be away for a while" option? Please advise.

**Comment by multimokia** [comment, marked as answer]
There's no limit to `I'm going to take you somewhere`, she's more than happy to just be with you. Happy travels!
